client/src/pages/Interview.js

```diff
@@ -98,6 +102,30 @@ const Interview = (props) => {
 		props.history.push('/dashboard');
 	};
 
+	const compileCode = async () => {
+		let extension = 'js';
+		switch (language) {
+			case 'python':
+				extension = 'py';
+				break;
+			case 'java':
+				extension = 'java';
+				break;
+			default:
+				extension = 'js';
```

Comment:
small note but this is redundant. `extension` is already `'js'` here. We can change the declaration to `let extension;`, or just remove this line and make it break on `default`.

Reply:
Yeah, I thought it was weird to have the 'js' assignation twice. I've already made the change, thanks for the suggestion.